server/index.js

```diff
@@ -1,9 +1,13 @@
+const newrelic = require('newrelic');
 const express = require('express');
 const parser = require('body-parser');
-const models = require('./models/index.js');
+const models = require('./models/pgModel.js');
+
 
 const port = 3333;
 const app = express();
+app.locals.newrelic = newrelic;
```

Comment:
What does this do? I added newrelic too, but I didn't see this in the docs

Reply:
it was from the browser setup, not sure if it's doing anything in my setup: https://docs.newrelic.com/docs/agents/nodejs-agent/supported-features/new-relic-browser-nodejs-agent

Reply:
Ahh right. I forgot we set up different ones. Got it